### ISSUE TITLE
Basic region assignment not working?

### ISSUE THREAD

**Issue by weebsplat** [Haoming02/sd-forge-couple#81]
Noticed that the extension seems to just not do it's thing when using basic region prompting. No problem when using advanced, but sometimes i just wanna go the quick easy route.

Running on the latest build of reForge.

**Comment by Haoming02** [comment]
Well, any example would be nice

**Comment by weebsplat** [comment]
Sorry about that. Was looking to get an example though and I think I realised the problem. The model I use works best with an appended string of text at the end of the prompt on a new line, so for ease-of-use I set up my webui to append it using a style once i hit generate and I have that function enabled by default on startup. that obviously has the issue of messing with the automatic region dividing of basic but doesn't interfere with the predetermined linecount for advanced assignment.

Will include some examples anyhow here (same prompt and seed), but TLDR: My bad, was being stupid. Sorry.

Prompt:
```
2girls, ayanami rei, plugsuit, classroom 
2girls, souryuu asuka langley, plugsuit, classroom 
```

...With the prompt changing to looking like this when style is applied:
```
2girls, ayanami rei, plugsuit, classroom
2girls, souryuu asuka langley, plugsuit, classroom, ,

masterpiece, best quality, amazing quality, very aesthetic, absurdres, newest, (volumetric lighting)

Negative prompt: (3d, censored), (worst quality, low quality), realistic, text,
```

Basic w/ applied style:
![00000-434981183](https://github.com/user-attachments/assets/1a93e78a-5919-4c38-8516-8653f1fc38a6)


Basic w/o applied style:
![00001-434981183](https://github.com/user-attachments/assets/cc925e76-a49c-4ee5-a08d-d66414a06e41)


Advanced w/ applied style:
![00002-434981183](https://github.com/user-attachments/assets/d725978d-7144-4faf-b583-41c5756705c6)

Advanced w/o applied style:
![00003-434981183](https://github.com/user-attachments/assets/c803cc45-70cb-40e5-944f-9221e2567fa8)

But yeah, I missed the most obvious solution to this somehow in my own time until now after already putting it up here, so sorry about that. Will close the issue now since it wasn't really an issue in the first place after all.
